core/arch/x86/decode_table.c

```diff
@@ -1498,6 +1498,7 @@ const instr_info_t * const op_instr[] =
 #define Wed TYPE_W, OPSZ_16_vex32_evex64
 #define Vex TYPE_V, OPSZ_16_vex32_evex64
 #define Wex TYPE_W, OPSZ_16_vex32_evex64
+#define Weh_x TYPE_W, OPSZ_half_16_vex32_evex64
```

Comment:
wrong code: should be `Wh_e`

Reply:
So we call '_x' either 16, vex32 and _e 16, vex32, or evex64? To me the '_x' meant 'any vector' while the 'e' addition to the type's size meant evex. We had discussed this and came up with Wex etc. It's all subject to interpretation that's why I wouldn't call it "wrong". I personally actually don't care whether it is Weh_x or Wh_e. So let's use '_e' for as elaborated here, but there may be other corner case names coming up later.

Reply:
Renamed.

Reply:
?  We had already agreed on what these mean.  `n_m` means "n out of m".  `x` is Intel's code, and it means "dq or qq".  `e` is our new size code for "dq or qq or oq".

Reply:
There are also two-letter codes following Intel's {p,s,v}{s,d,i} that indicate the element type of a vector.  We decided to extend the {p,s,v} to add an 'e' to mean, again, 128, 256, or 512.  So e{s,d,i} is a valid code for a packed vector, though not all that useful to DR since the IR doesn't track the element type.